pkg/apis/crds/karpenter.sh_machines.yaml

```diff
@@ -0,0 +1,337 @@
+---
```

Comment:
Can we check this in separately?

Reply:
Curious why you are thinking that it should be in separately. I get that it reduces the amount of lines in the PR but the PR doesn't work without the CRD. IMO a PR should produce a valid build after it's merged. Without this change, that isn't the case